package.json

```diff
@@ -9,6 +9,7 @@
         "@livechat/customer-sdk": "^3.1.0",
         "@loadable/component": "^5.15.2",
         "@svgr/webpack": "^6.1.2",
+        "antd": "^4.17.4",
```

Comment:
Why are we using Ant Design? We have our input components, I think re-using them would be enough.